static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -1,25 +1,25 @@
-.content--immersive {
-    .content__headline {
-        @include fs-headline(9);
-        font-weight: 200;
-        line-height: 1;
-        padding-top: $gs-baseline / 2;
-        padding-bottom: 1.5rem;
+.content__headline--immersive {
+    @include fs-headline(9);
+    font-weight: 200;
+    line-height: 1;
+    padding-top: $gs-baseline / 2;
+    padding-bottom: 1.5rem;
 
-        @include mq(desktop) {
-            font-size: 4rem;
-        }
+    @include mq(desktop) {
+        font-size: 4rem;
     }
+}
 
-    .content__headline--with-main-media {
-        @include fs-headline(7, true);
-        line-height: 1;
+.content__headline--immersive--with-main-media {
```

Comment:
I guess it means to say "this modifier is a subclass of another modifier". It only makes sense to use modifier `--x--y` when also using `--x`.